app/liquid_tags/twitch_tag.rb

```diff
@@ -0,0 +1,37 @@
+class TwitchTag < LiquidTagBase
+  PARTIAL = "liquids/twitch".freeze
+
+  def initialize(_tag_name, slug, _parse_context)
+    super
+    @url = parsed_url(SiteConfig.app_domain)
+    @slug = parsed_slug(slug)
+    @width = 710
+    @height = 399
+  end
+
+  def render(_context)
+    ApplicationController.render(
+      partial: PARTIAL,
+      locals: {
+        url: @url,
+        slug: @slug,
+        width: @width,
+        height: @height
+      },
+    )
+  end
+
+  private
+
+  # Strip out port number because it confuses Twitch
+  def parsed_url(url)
+    url.split(":")[0]
+  end
```

Comment:
I was going to suggest using `ApplicationConfig.app_domain_no_port` but that's tied to the environment variable. Let's keep this here, I'll add a `SiteConfig.app_domain_no_port` in a separate PR not to derail this one :)